templates/iam_policy/get_latest_preservica_version_lambda_policy.json.tpl

```diff
@@ -0,0 +1,38 @@
+{
+  "Statement": [
+    {
+      "Action": [
+        "dynamodb:BatchGetItem"
+      ],
+      "Effect": "Allow",
+      "Resource": "${dynamo_db_arn}",
+      "Sid": "readWriteDynamoDB"
```

Comment:
The permissions are only for read so can you change the Sid to `readDynamoDb` please.

Reply:
What do we want to happen when Preservica update their demo/reltest system? If we're only writing to ddb this Lambda will continue to generate events on a new version until someone changes the value in ddb.

Reply:
Isn't that the idea, when they update their test system, we manually update the value in Dynamo. That seems safer than getting the lambda to do it because we might miss the initial message.